viewing_party/party.py

```diff
@@ -1,23 +1,180 @@
 # ------------- WAVE 1 --------------------
 
+from tests.test_constants import MOVIE_TITLE_1
+
+movie_dictionary = {}
+
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        movie_dictionary["title"] = title
+        movie_dictionary["genre"] = genre
+        movie_dictionary["rating"] = rating
+        return movie_dictionary
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    user_data = {"watched":[movie]}
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data = {"watchlist": [movie]}
+    return user_data
+
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)
+            return user_data
+    else:
+            return user_data

```

Comment:
The `user_data` is returned at the end of this function whether or not a change is made to it.  I recommend pulling this out of the logic.
```suggestion
    return user_data
    
```